include/nbl/video/alloc/SubAllocatedDescriptorSet.h

```diff
@@ -285,6 +307,12 @@ class SubAllocatedDescriptorSet : public core::IReferenceCounted
 			//
 			// this needs to be done as a IGPUDescriptorSet::nullify(const uint32_t binding, 
 			// std::span<uint32_t> indices) function + a virtual nullify_impl
+			video::IGPUDescriptorSet::SDropDescriptorSet dropDescriptorSet;
+			dropDescriptorSet.dstSet = m_descriptorSet.get();
+			dropDescriptorSet.binding = binding;
+			dropDescriptorSet.arrayElement = i;
+			dropDescriptorSet.count = 1;
+			m_logicalDevice->nullifyDescriptors({ &dropDescriptorSet, 1 });
```

Comment:
there's a way to make this slightly more efficient, which is to do what I tell you NOT to do in the `multi_allocate` which is to take the `SDropDescriptorSet` (which should be called `SNullifyDescriptors`) pointer from outside, and just write out the dops here and submit the `nullify` outside

This would allow you to do just one call to `nullifyDescriptors` in `cull_frees` and `try_multi_allocate`

You'd obviously need an override that calls it automagically if no arguments supplied